src/plugins/pbjx/components/history/index.jsx

```diff
@@ -47,21 +50,26 @@ class History extends React.Component {
     const {
       delegate,
       events,
+      isEditMode,
       getUser,
       getHistoryRequestState: { response, status, exception },
-      streamId,
-      schema,
+      formName,
+      node,
     } = this.props;
 
     return (
       <EventStream
         events={events}
         getUser={getUser}
+        formName={formName}
```

Comment:
@rsumilang doesnt look like this prop is used by EventStream

Reply:
I removed formName and node but it's a little weird because I then have to remove them from the required props. However they are required because the delegate needs them but now it's not listed anywhere. Hmm.

Reply:
@rsumilang its a wrinkle of required props, not that big of a deal